staging/src/k8s.io/apiextensions-apiserver/pkg/apiserver/conversion/nop_converter.go

```diff
@@ -61,6 +61,15 @@ func (c *nopConverter) Convert(in, out, context interface{}) error {
 	if err != nil {
 		return err
 	}
+	// If the context is a groupVersener passed from [this](https://bit.ly/2CCByRM), we should convert to that
+	// version.
+	if context != nil {
+		if groupVersioner, ok := context.(runtime.GroupVersioner); ok {
```

Comment:
this interacts oddly with outGVK. It's not clear which should take priority if they disagree.